### ISSUE TITLE
Error: Can't access Form Settings

### ISSUE THREAD

**Issue by margenstepa** [jan-vince/smallcontactform#160]
Hi, I'm getting an error while accessing the Contact Form Settings from the backend dashboard (/backend/system/settings/update/janvince/smallcontactform/settings).

Tried to fix it by changing the janvince_smallcontactform_settings item on the system_settings table from the DB but nothing happened and also reinstalled the whole package multiple times with no success on solving the error. 

<img width="1380" alt="Screenshot 2023-04-19 at 12 06 29 PM" src="https://user-images.githubusercontent.com/65454034/233042306-b6334672-6e64-43a4-ab0f-d00a830fe070.png">

**Comment by jan-vince** [comment]
Is this on new installation or you have edited plugin's settings before the error?

**Comment by margenstepa** [comment]
Fresh installation. Also reinstalled the plugin like 3 times.

**Comment by jan-vince** [comment]
Check your database:

- table: janvince_smallcontactform_settings
- row where column item has value janvince_smallcontactform_settings
- in column value look for field record for `form_fields` (it's JSON)

`form_fields` should be an array

**Comment by margenstepa** [comment]
Solved! I tried many different forms of the form_fields (array, object, null) and it didn't worked. But then... I cleared cache and the error went away... 
If I remember correctly the error started when I tried to create a new field on the form from the form settings and refreshed the page without saving which seemed very weird that I got the error afterwards since I didn't save anything.

Thanks for you help and great plugin, I use it a lot :)
